Day2/REST.txt

```diff
@@ -0,0 +1,42 @@
+1)
```

Comment:
For This, you need to just show how the request and response for resful api would be: 
for eg something like below:
Name: get a list of books
specify the path eg: base-url/books
possible status codes eg
404: route not found
200: success
401: unauthenticatied and so on
Write the format of how would the response look like on success and failure with status codes